hawkular-bus/pom.xml

```diff
@@ -79,28 +69,11 @@
       </dependency>
 
       <dependency>
-        <groupId>org.wildfly.core</groupId>
-        <artifactId>wildfly-controller</artifactId>
-        <version>${version.org.wildfly.core}</version>
-      </dependency>
-
-      <dependency>
         <groupId>org.wildfly</groupId>
-        <artifactId>wildfly-dist</artifactId>
+        <artifactId>wildfly-jms-client-bom</artifactId>
```

Comment:
I'm sure this dependency is correct, but just to confirm: isn't this also included on the main wildfly-javaee bom?